packages/main/src/Toolbar.ts

```diff
@@ -128,6 +129,15 @@ class Toolbar extends UI5Element {
 	@property({ defaultValue: "" })
 	accessibleNameRef!: string;
 
+	/**
+	 * Defines the toolbar design.
+	 * @public
+	 * @default "Auto"
+	 * @since 2.0.0
+	 */
+	@property({ type: ToolbarDesign, defaultValue: ToolbarDesign.Auto })
```

Comment:
I'm not sure that there is difference between auto and solid overall. Also we don't have any specific usage where toolbar is changed due the auto value

Reply:
As we discussed, the "Auto" value was redundant because it was identical to "Solid." The "Auto" is removed and the default value is set  to "Solid," ensuring a consistent visual appearance.